spec/order_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'order'
+
+describe "Order" do 
+
+  let (:menu){double :menu, dishes: {biriyani: 5.00, uk_curry: 6.00, cheeseburger: 4.00, pizza: 5.00, side_of_salmonella: 1.95}}
```

Comment:
I would extract the value of `dishes` to a private method or other `let` statement